rust/src/delta.rs

```diff
@@ -605,6 +608,29 @@ impl DeltaTable {
         self.state.min_writer_version
     }
 
+    /// Run the dry run of the Vacuum command on the Delta Table: list files no longer referenced by a Delta table and are older than the retention threshold.
+    pub fn vacuum_dry_run(&self, retention_hours: u64) -> Result<Vec<String>, DeltaTableError> {
+        let retention_checked = match retention_hours {
+            _ if retention_hours < 168 => {
```

Comment:
Could you copy the warning text from the official doc to explain the recommended minimal 7 days retention magic number?